src/datreant/core/collections.py

```diff
@@ -439,6 +439,17 @@ def relpaths(self):
         return [member.relpath for member in self]
 
     @property
+    def parents(self):
```

Comment:
If the ``View`` contains ``Tree``s or ``Leaf``s with different parent directories, then yes. A ``View`` can have any collection of ``Tree`` and ``Leaf`` objects as members, from anywhere in the filesystem, existing or not.